packages/components/src/pages/VsCodeExtension.vue

```diff
@@ -597,6 +597,15 @@ export default {
                   nodes.add(event.id);
                 }
               });
+            } else {
+              this.filteredAppMap.events.forEach((event) => {
+                if (
+                  event.isCall() &&
+                  event.toString().toLowerCase().includes(item.toLowerCase())
+                ) {
+                  nodes.add(event.id);
+                }
+              });
```

Comment:
Please add a test case for this

Reply:
@dustinbyrne , I already added string `#link_to_edit_url` to filter input in test case

https://github.com/applandinc/appmap-js/pull/492/files#diff-f5e13ae703840c989eb60ca988f5d4e016a49c59a6e4366c0cdeead459043f88R710